From 5760c6b1a3c9b6dd8e70c9d1ea78fcf192250b4a Mon Sep 17 00:00:00 2001
From: Shengjiu Wang <shengjiu.wang@freescale.com>
Date: Mon, 16 Mar 2015 15:30:11 +0800
Subject: [PATCH] Revert "launch: Avoid specifically starting PA and rely on
 autospawn/socket activation"

This reverts commit 42156d2b5ac797e5f28f8d0d38b691053f3f6fc7.

Upstream status: Inappropriate [i.MX specifiec] 

Signed-off-by: shengjiu.wang <b02247@freescale.com>
---
 src/daemon/start-pulseaudio-x11.in |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/daemon/start-pulseaudio-x11.in b/src/daemon/start-pulseaudio-x11.in
index 16e46b1..e108645 100755
--- a/src/daemon/start-pulseaudio-x11.in
+++ b/src/daemon/start-pulseaudio-x11.in
@@ -17,6 +17,8 @@
 
 set -e
 
+@PA_BINARY@ --start "$@"
+
 if [ x"$DISPLAY" != x ] ; then
 
     @PACTL_BINARY@ load-module module-x11-publish "display=$DISPLAY" > /dev/null
-- 
1.7.9.5

